### PR TITLE
Add getting started guide

### PR DESCRIPTION
**Pull request by udiWertheimer**

### CODE REVIEW

File: getting_started.md [hunk 1]
```diff
@@ -1 +1,339 @@
+# Getting Started with ColoredCoins
 
+Thank you for your interest in ColoredCoins! In this guide, we will explain how to quickly set up and start using the open source ColoredCoins SDK.
+
+The SDK is meant to be used in your own apps, in order to enable creating and transacting digital assets using the ColoredCoins protocol. It's a JavaScript SDK, which can be used in a browser or in a Node.js app. You can use the SDK in a client app, allowing all users to manipulate their tokens securely with their own keys, or you can deploy it on a server-side app and manage transactions on your trusted server.
```
Comment (oleiba) [inline getting_started.md]: link to our colored-coins specification?

File: getting_started.md [hunk 2]
```diff
@@ -1 +1,339 @@
...
+npm init
+```
+And answer all the questions.
+
+Finally, we'll add the ColoredCoins SDK to our package.json file and install:
+```
+npm install --save coloredcoins-sdk
+```
+
+By default, the SDK is set up to use our Community-Hosted Node of ColoredCoins Core, which validates transactions and allows inspecting the state of the network. So there's no need to install anything else, you can start coding! After you gain some confidence with the SDK, we encourage you to install your own instance of ColoredCoins Core in a controlled environment and use that, for improved security.
```
Comment (tal-beja) [inline getting_started.md]: >  own instance of ColoredCoins Core

add link to the docker repo and later to the CC full node

File: getting_started.md [hunk 3]
```diff
@@ -1 +1,339 @@
...
+
+Let's create a file named `index.js` and open that in your favorite editor.
+We'll start by initializing the SDK object, specifying your *mnemonic* and *network*.
+
+### What's a mnemonic?
+
+ColoredCoins is using the Bitcoin network, and in Bitcoin each stakeholder in the system has an identity, which is determined by a key which has two parts, a public and a private part. The private key must be kept secret and safe, you can think of it as a password. From that private key, a matching public key is created and you can think of it as your address, or username, which others will use to send funds to you. You can safely share that address.
+
+The "mnemonic" is a random list of 12 words that will be automatically by the SDK, representing a *private seed*, from which many *private keys* are created, each with its own *public address*. We allow multiple addresses per user for flexibility, but you can think of the *mnemonic* as a private password that represents an identity in the system.
+
```
Comment (tal-beja) [inline getting_started.md:39]: I would add here links to BIPs: 32 , 44 (hdkeys) and 39 (mnemonic)

File: getting_started.md [hunk 4]
```diff
@@ -1 +1,339 @@
...
+### What's a network?
+
+Bitcoin transactions have a cost attached - miner fees. These can range from a few cents to a Dollar or more, based on current traffic in the network. To allow development and testing without paying those fees, there's a "test network" for Bitcoin, called `testnet` for short. This network isn't secure for real-value transactions, but you can transact for free. Transactions still have fees, but the fees are paid in "test bitcoins", which you can get for free from "Testnet Bitcoin Faucets", websites that will send you free test bitcoins to help with development.
+
```
Comment (tal-beja) [inline getting_started.md]: > Testnet Bitcoin Faucets

links to faucets

**Review by tal-beja — COMMENTED** [review]
Looks good!
Loved the Lala land story
:+2:
(two thumbs up)

**Pull request merged**